src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,62 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+
+public class Intake extends SubsystemBase {
+    public static Intake INSTANCE;
+    private final WPI_TalonSRX motor = new WPI_TalonSRX(Ports.Intake.MOTOR_PORT);
+    private final Solenoid solenoid = new Solenoid(PneumaticsModuleType.CTREPCM, Ports.Intake.SOLENOID_PORT);
+
+    public Intake() {
+        motor.setInverted(Ports.Intake.MOTOR_INVERTED);
+    }
+
+
+    /**
+     * lazy instantiation
+     *
+     * @return the subsystem instance
+     */
+    public static Intake getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new Intake();
+        }
+        return INSTANCE;
+    }
+
+    /**
+     * sets the power of the intake
+     *
+     * @param power desired power

```

Comment:
add period at the end of the sentence and first letter should be capital (and in any other documentation)